content/academy/dynamic_pages_and_spas.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Dynamic pages and SPAs
+title: API Scraping
 description: Learn all about how the professionals scrape various types of APIs with various configurations, parameters, and requirements.
 menuWeight: 3
 category: courses
 paths:
 - dynamic-pages-and-spas
 ---
 
-# [](#dynamic-pages-and-spas) Dynamic pages and single-page applications
+# [](#dynamic-pages-and-spas) Dynamic Pages and Single-Page Applications (SPAs)
```

Comment:
Please, use sentence case case. The capitalising for `single-page applications` is also unnecessary - it's just a type of app, not a proper name as far as I'm aware